### DISCUSSION TITLE
How to propose documentation changes

### DISCUSSION THREAD

**Discussion by egberts** [getpelican/pelican#3347] [General]
I couldn't find anything in the way of making corrections to the readdoc of Pelican over at https://docs.getpelican.com/en/latest/contribute.html

What are the procedures for doing this?

**Comment by justinmayer** [comment]
I don't understand. Do you mean _ReadTheDocs_? What are examples of corrections that you would propose?

**Comment by egberts** [comment]
Yes, thank you for the clarification:  I meant, ReadTheDocs.  I probably got the 'readdoc' from a subdirectory of its website URL.

What are the channels, venue, discussion, forum route for one to take in submitting updates to ReadTheDocs?  That part does not seem to be clearly posted.

**Comment by justinmayer** [comment]
ReadTheDocs is merely the back-end service that publishes the Pelican documentation to https://docs.getpelican.com. So if I understand you correctly, you are actually asking, "How does one make and submit proposed changes to the documentation?"

If that is indeed what you are asking, then the answer is already in the documentation: https://docs.getpelican.com/en/latest/contribute.html#building-the-docs

The slightly-longer answer is:

1. `invoke docserve`
2. Make changes to documentation in `docs/`
3. Go to http://localhost:8000 and review changes in local browser
4. If everything looks good in local browser, submit PR with proposed documentation changes

**Comment by egberts** [comment]
Got it.... `docs/`.   My apology on the denseness.  Just connecting the dots between various websites to various subdirectories.

Thank you.
